person.rb

```diff
@@ -1,21 +1,29 @@
-class Person
+require './nameable'
+require './decorator'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permission: true)
-    @id = Random.rand(1..500)
+    super()
+    @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
   end
 
   def can_use_services?
-    of_age? || @parent_permission
+    of_age? || parent_permission ? true : false
+  end
+
+  def correct_name
+    @name
   end
 
   private
 
   def of_age?
-    @age >= 18
+    age.to_i >= 18.to_i
   end
 end
```

Comment:
- [ ] `*A method for adding rentals should be present in the Person class

`def add_rental` should take (book, date) as an argument.